src/components/ButtonPanel.js

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+
+class ButtonPannel extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {};
+  }
+
+  render() {
+    return (
+      <div className="App">
+        <p>
+          ButtonPannel component connected
+        </p>
```

Comment:
[Optional]

Let's remove this as we plan to make it look as similar as possible to the picture in the requirement https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/44896082/

   ![image](https://user-images.githubusercontent.com/36057474/73360662-bdd24d00-42a3-11ea-92ad-d0c4c7079297.png)